src/main/java/nl/novi/techiteasy1121/services/TelevisionService.java

```diff
@@ -1,72 +1,88 @@
 package nl.novi.techiteasy1121.services;
 
-import nl.novi.techiteasy1121.Dtos.TelevisionDto;
-import nl.novi.techiteasy1121.Dtos.TelevisionInputDto;
+import nl.novi.techiteasy1121.dtos.TelevisionDto;
+import nl.novi.techiteasy1121.dtos.TelevisionInputDto;
 import nl.novi.techiteasy1121.exceptions.RecordNotFoundException;
 import nl.novi.techiteasy1121.models.Television;
+import nl.novi.techiteasy1121.repositories.CIModuleRepository;
+import nl.novi.techiteasy1121.repositories.RemoteControllerRepository;
 import nl.novi.techiteasy1121.repositories.TelevisionRepository;
 import org.springframework.stereotype.Service;
 import org.springframework.web.bind.annotation.RequestBody;
 
 import java.util.ArrayList;
 import java.util.List;
-import java.util.Optional;
 
-// Zet de annotatie boven de klasse, zodat Spring het herkent en inleest als Service.
 @Service
 public class TelevisionService {
 
-    // We importeren de repository nu in de service in plaats van in de controller.
-    // dit mag met constructor injection of autowire.
     private final TelevisionRepository televisionRepository;
 
-    public TelevisionService(TelevisionRepository televisionRepository){
+    private final RemoteControllerRepository remoteControllerRepository;
+
+    private final RemoteControllerService remoteControllerService;
+
+    private final CIModuleRepository ciModuleRepository;
+
+    private final CIModuleService ciModuleService;
+
+    public TelevisionService(TelevisionRepository televisionRepository,
+                             RemoteControllerRepository remoteControllerRepository,
+                             RemoteControllerService remoteControllerService,
+                             CIModuleRepository ciModuleRepository,
+                             CIModuleService ciModuleService
+                             ){
         this.televisionRepository = televisionRepository;
+        this.remoteControllerRepository = remoteControllerRepository;
+        this.remoteControllerService = remoteControllerService;
+        this.ciModuleRepository = ciModuleRepository;
+        this.ciModuleService = ciModuleService;
     }
 
-    // Vanuit de repository kunnen we een lijst van Televisions krijgen, maar de communicatie container tussen Service en
-    // Controller is de Dto. We moeten de Televisions dus vertalen naar TelevisionDtos. Dit moet een voor een, omdat
-    // de translateToDto() methode geen lijst accepteert als argument, dus gebruiken we een for-loop.
     public List<TelevisionDto> getAllTelevisions() {
         List<Television> tvList = televisionRepository.findAll();
-        List<TelevisionDto> tvDtoList = new ArrayList<>();
-
-        for(Television tv : tvList) {
-            TelevisionDto dto = transferToDto(tv);
-            tvDtoList.add(dto);
-        }
-        return tvDtoList;
+        return transferTvListToDtoList(tvList);
```

Comment:
Zorg ervoor dat de methoden niet te lang worden, overweeg om complexe logica op te splitsen in kleinere methoden.